### PR TITLE
fix: 🐛 add missing lightbulb16

### PR DESCRIPTION
**Pull request by kmelkon**

### CODE REVIEW

File: src/Atoms/Icon/communication/lightbulb/Lightbulb16.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { IconBase } from '../../IconBase';
+import { IconProps } from '../../IconBase.types';
+
+const Lightbulb24: React.FC<IconProps> = (props) => {
```
Comment (alvelid) [inline src/Atoms/Icon/communication/lightbulb/Lightbulb16.tsx:6]: 16

File: src/Atoms/Icon/communication/lightbulb/Lightbulb24.tsx [hunk 2]
```diff
@@ -3,9 +3,9 @@ import React from 'react';
 import { IconBase } from '../../IconBase';
 import { IconProps } from '../../IconBase.types';
 
-const Lightbulb24: React.FC<IconProps> = (props) => {
+const Lightbulb16: React.FC<IconProps> = (props) => {
```
Comment (alvelid) [inline src/Atoms/Icon/communication/lightbulb/Lightbulb24.tsx:6]: 24

File: src/Atoms/Icon/communication/lightbulb/Lightbulb16.tsx (hunk 1 above)
Comment (kmelkon) [reply src/Atoms/Icon/communication/lightbulb/Lightbulb16.tsx:6]: nice

File: src/Atoms/Icon/communication/lightbulb/Lightbulb24.tsx [hunk 3]
```diff
@@ -3,9 +3,9 @@ import React from 'react';
...
   return (
-    <IconBase {...props} width={24} height={24}>
+    <IconBase {...props} width={16} height={16}>
```
[the remark this answers is not in the document]
Comment (kmelkon) [reply src/Atoms/Icon/communication/lightbulb/Lightbulb24.tsx:8]: 24 :P

**Review by alvelid — APPROVED** [review]

**Pull request merged**